README.md

```diff
@@ -5,19 +5,51 @@ A flask-based backend for FertiScan.
 ## Setup for Development
 
 ### Prerequisites
+
 - Python 3.8 or higher
 - [pip](https://pip.pypa.io/en/stable/installation/)
-- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+- [virtualenv](https://virtualenv.pypa.io/en/latest/installation.html)
 - Azure Document Intelligence and OpenAI API keys
 
 ### Running
 
+#### Locally
+
 ```sh
 cd fertiscan-backend
 pip install -r requirements.txt
 python ./app.py
 ```
 
+#### With docker
+
+1. Build the docker image
+
+    ```bash
+    docker build -t fertiscan-backend \
+    --build-arg ARG_AZURE_API_ENDPOINT=your_actual_azure_form_recognizer_endpoint \
+    --build-arg ARG_AZURE_API_KEY=your_actual_azure_form_recognizer_key \
+    --build-arg ARG_AZURE_OPENAI_API_ENDPOINT=your_actual_azure_openai_endpoint \
+    --build-arg ARG_AZURE_OPENAI_API_KEY=your_actual_azure_openai_key \
+    --build-arg ARG_PROMPT_PATH=actual_path/to/prompt_file \
+    --build-arg ARG_UPLOAD_PATH=actual_path/to/upload_file \
+    .
```

Comment:
Same as the previous one. Add this [variable](https://github.com/ai-cfia/fertiscan-backend/blob/4c5a11fe51e0703566989be547c04e240057962d/.env.template#L12).

Reply:
Good I will also update the README.md